target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,17 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/airtight_c-65
```

Comment:
Device recipes should be sorted by alphabetic order.

Reply:
I did not put Seama on top. The rest is alphabetic.

Reply:
The seama is common define used by few devices so yes, it should be on above them, but definition for Your device is above `8dev_carambola2` and digits should be before letters.

Then fix commit title, SoB line and rebase on top of master and that should be it.
Good luck.